perun-core/src/main/java/cz/metacentrum/perun/core/impl/ResourcesManagerImpl.java

```diff
@@ -153,7 +155,7 @@
 				}
 				// fetch each resource tag and add it to rich resource
 				ResourceTag tag = RESOURCE_TAG_MAPPER.mapRow(rs, rs.getRow());
-				if (tag != null && tag.getId() != 0) {
+				if (tag != null && tag.getId() != 0 && myObject != null) {
```

Comment:
So there is no reason to call: `RESOURCE_TAG_MAPPER.mapRow(rs, rs.getRow())` if myObject (richResource) is null. Then you can always call "continue" when myObject is null there. Maybe also log it, because it is weird.